pipeline_manager/scripts/delete_project.sh

```diff
@@ -9,4 +9,4 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 
-curl -X POST http://localhost:8080/delete_project  -H 'Content-Type: application/json' -d '{"project_id":'$1'}'
+curl -X DELETE http://localhost:8080/projects -H 'Content-Type: application/json' -d '{"project_id":'$1'}'
```

Comment:
For consistency, it's better to do a delete to `/projects/{project_id}`. Similar issues with delete for pipelines and configs. Given that we already have URLs like `/projects/$1/{config, pipelines}` etc elsewhere.

Reply:
Otherwise, what we're suggesting with the API is a way to delete a collection (e.g. multiple projects), in which case the payload should be able to state an array of IDs to delete. This is also fine, just want to be sure what the API is.